### PR TITLE
[NL] Installation, Interface, Main Page, Registration, and Welcome

### PR DESCRIPTION
**Pull request by KevKjef**
The start of my translation of the wiki to dutch, also includes fixes some minor mistakes in the english pages.
Finished translating: main_page, welcome, registration, installation and interface to dutch
Note: part of the interface is outdated so I updated it.
A different part that is also outdated (final part of interface) will probably need some updating aswell, since the results screen changed, but I left this alone for now.

Second note: I translated the wiki to dutch except for a couple words/sentences. I think referring to groups ,mods and a handful of other things in their english variant sounds a lot better and makes more sense. I might want to change things of the game to dutch as the local leaderboards and a variety of other sections got translated.

### CODE REVIEW

**Comment by KevKjef** [comment]
It seems I uploade the interface both here and separately, sorry for this im still figuring out how things work ;.;

File: wiki/Registration/nl.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+   - In het veld, Password, maak je een wachtwoord aan. Je wachtwoord moet ten minste bestaan uit 8 karakters.
+3. Als je alles hebt ingevuld, klik je op `1. Create my account!`. Je wordt dan automatisch ingelogd.
+
+## Wat nu?
+
+Vergeet niet de [regels](/wiki/Rules) zorgvuldig te lezen!
+
+Als je dit hebt gedaan, kan jouw ritmisch avontuur beginnen! Je kunt [beatmaps toevoegen](/wiki/Installation#adding-beatmapsets) om te spelen, [een skin maken](/wiki/Skinning) of zelf [een beatmap maken](/wiki/Beatmapping). Ook kun je bij de [Introducties subforum](https://osu.ppy.sh/community/forums/8) langsgaan om "Hallo" te zeggen.
+
+Mocht je nog meer hulp nodig hebben, kun je hierom vragen in het `#help` [Internet Relay Chat](/wiki/Internet_Relay_Chat) (IRC) kanaal, na het openen van de chat console (Druk op `F8` of `F9` in-game) en type vervolgens `/join help`. Ook kun je een post maken met jouw vraag in de [Help subforum](https://osu.ppy.sh/community/forums/5).
```
Comment (nwatti) [inline wiki/Registration/nl.md:21]: Je kunt ook een post maken met jouw vraag op het [Help subforum](https://osu.ppy.sh/community/forums/5).

File: wiki/Welcome/nl.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Welkom
+
+Welkom bij osu!, een free-to-win rhythm game ontworpen door peppy. Osu! bevat vier game modes: osu!standard, een simulator waarin je op cirkels klikt; osu!taiko, een drum emulator; osu!catch, een fruit vanger; en osu!mania, een synthesizer waarin je op toetsen in slaat.
```
Comment (nwatti) [inline wiki/Welcome/nl.md:3]: en osu!mania, een synthesizer waarin je op toetsen ramt.

File: wiki/Welcome/nl.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+Modding is het proces waar gebruikers (In het algemeen genaamd "Modders") de beatmap van iemand anders beoordelen. Modding speelt een grote rol in de kwaliteitscontrole, aangezien makers hierdoor de mogelijkheid krijgen problemen van hun beatmap op te lossen.
+
+## Skinning
+
+*Hoofdpagina: [Skinning](/wiki/Skinning)*
+
+*Zie ook: [Skinning Tutorial](/wiki/Skinning_Tutorial)*
+
+Skinning geeft iedereen de mogelijkheid om het uiterlijk van osu! te veranderen. Dit kunnen kleine veranderingen zijn, zoals het veranderen van de cursor of grote veranderingen, zoals het uiterlijk van alle gamemodes en interfaces aanpassen.
```
Comment (nwatti) [inline wiki/Welcome/nl.md:41]: Skinning geeft iedereen de mogelijkheid om het uiterlijk van osu! te veranderen. Dit kunnen kleine veranderingen (zoals het veranderen van de cursor) of grote veranderingen (zoals het uiterlijk van alle gamemodes en interfaces aanpassen) zijn.

File: wiki/Main_Page/nl.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
+---
+layout: main_page
+---
+
+<!-- Do not add any empty lines inside this div. -->
+<div class="wiki-main-page__blurb">
+Welkom op de osu! wiki. Dit is een project met een groot aanbod van gerelateerde informatie over osu!
```
[the remark this answers is not in the document]
Comment (KevKjef) [reply wiki/Main_Page/nl.md:7]: Ik vind de 'aan' hier niet passen.

File: wiki/Registration/nl.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# Registratie
+
+*Waarshuwing!: Het hebben van meer dan een osu! account op elk moment is een overtreding van de [osu! regels](/wiki/Rules)!*
+
+1. Na het [installeren](/wiki/Installation) van osu! en het uitvoeren hiervan, zal osu! je vragen om in te loggen of jezelf te registreren. Ervan uitgaande dat je hier bent, omdat je een account nodig hebt, klik op `Create an account`.
```
[the remark this answers is not in the document]
Comment (KevKjef) [reply wiki/Registration/nl.md:5]: , omdat ??

File: wiki/Welcome/nl.md [hunk 6]
```diff
@@ -38,7 +38,7 @@ Modding is het proces waar gebruikers (In het algemeen genaamd "Modders") de bea
 
 *Zie ook: [Skinning Tutorial](/wiki/Skinning_Tutorial)*
 
-Skinning geeft iedereen de mogelijkheid om het uiterlijk van osu! te veranderen. Dit kunnen kleine veranderingen zijn, zoals het veranderen van de cursor of grote veranderingen, zoals het uiterlijk van alle gamemodes en interfaces aanpassen.
+Skinning geeft iedereen de mogelijkheid om het uiterlijk van osu! te veranderen. Dit kunnen kleine veranderingen zijn (zoals het veranderen van de cursor) of grote veranderingen (zoals het uiterlijk van alle gamemodes en interfaces aanpassen) zijn.
```
Comment (nwatti) [inline wiki/Welcome/nl.md:41]: eerste zijn moet weg

File: wiki/Interface/nl.md [hunk 7]
```diff
@@ -0,0 +1,304 @@
...
+![](img/osu!_online_score_updated2020.jpg "Voorbeeld van een osu!online score")
+
+Dit is jouw online leaderboard. Je kunt hier komen door naar beneden te scrollen op het resultatenscherm. De lokale rangschikking zal jouw naam en score tonen zoals gebruikelijk.
+
+1. Verandeer de naam van het record van de replay hier. Jouw spelersinformatie, je kunt hier jouw [PP](/wiki/Performance_Points), Globale Rang, Totale Score, Overall [Accuracy](/wiki/Accuracy), en level balk zien.
+2. Opties met betrekking tot de beatmap. Je kunt je favoriete beatmaps vinden op je profiel onder Beatmaps->Favourite Beatmaps.
+   1. Export Replay als `.osr`: Slaat de replay op zodat deze online kan worden gebruikt.
+   2. Online Favourite: Voegt de beatmap toe aan jouw favorieten. 
+3. Hier is het verschil te zien wat jouw score verandert aan je profiel. Je kunt hier zien hoeveel je bent geklommen of gedaald in de Globale Rangschikking, hoeveel score is toegevoegd aan je Totale Score en hoeveel je Gemiddelde Accuracy is verandert. De tekst vervaagt erg snel, dus mis het niet!
+4. Spreekt voor zich. Deze balk komt niet in beeld wanneer je offline speelt of de beatmap niet is geupload.
```
Comment (MegaApplePi) [inline wiki/Interface/nl.md:297]: It looks like you are missing the items under this point, lines 298 - 303 from the English version.

**Comment by TicClick** [comment]
@KevKjef @nwatti you may want to re-check the original articles for updates you want to reflect in the translations

**Comment by TicClick** [comment]
going to close the PR for now to hide it from a list of open pulls, please don't hesitate to reopen it or create a new one once you feel like contributing

**Pull request closed without merging**